### DISCUSSION TITLE
OTP was not sent failure

### DISCUSSION THREAD

**Discussion by venky586** [saran-surya/email_auth#27] [Q&A] [answered]
Hi, I was trying to test my flutter app using your pub package 'email_auth: ^1.0.0'.  I was able to get the OTPs on my test Email IDs, but after sometime it started throwing a console message 'OTP was not sent failure'. But when I updated the EmailAuth.sessionName to other name which was not used earlier the OTP mail was sent successfully. Can you please clarify on this whether session can be blocked from your end?

**Comment by saran-surya** [comment, marked as answer]
Hi @venky586,

As Mentioned in the package, the test servers have a limit of 30 mails per session, you will have to create a production server to match your production needs. 
Kindly check the note here : https://github.com/saran-surya/email_auth#features

To set up a production server, Kindly check this link : https://saran-surya.github.io/email-auth-node/

Happy coding 😸
